cmake/config/CMakeLists.txt

```diff
@@ -87,6 +87,56 @@ foreach(_build ${DEAL_II_BUILD_TYPES})
   list(APPEND CONFIG_TARGET ${_keyword} \${DEAL_II_TARGET_${_build}})
 endforeach()
 
+#
+# Collect feature configuration that will be added to the configuration files:
+#
+
+set(FEATURE_CONFIGURATION_STRING "
+#
+# Feature configuration:
+#
+
+set(DEAL_II_WITH_CXX11 ON)
+set(DEAL_II_WITH_CXX14 ON)
+set(DEAL_II_WITH_CXX17 ON)
+set(DEAL_II_WITH_CXX20 ${DEAL_II_HAVE_CXX20})
+set(DEAL_II_WITH_THREADS ON)\n"
+  )
```

Comment:
Do we still need to add those? Or is time to only export the `DEAL_II_HAVE_*` variants?

Reply:
I think for backwards compatibility it is important. Probaly all of my projects would break without these configuration options available :smile: